src/calculateRentalCost.js

```diff
@@ -1,10 +1,16 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
Remove the unnecessary upper bound check `days < 7` since the previous `if` statement already returns if `days >= 7`, and avoid adding `else` after `if` with `return` (checklist items 4 and 5).